build.sh

```diff
@@ -50,12 +44,22 @@ if [ "${POLICY}" == "manylinux2014" ]; then
 		LD_LIBRARY_PATH_ARG="${DEVTOOLSET_ROOTPATH}/usr/lib64:${DEVTOOLSET_ROOTPATH}/usr/lib:${DEVTOOLSET_ROOTPATH}/usr/lib64/dyninst:${DEVTOOLSET_ROOTPATH}/usr/lib/dyninst:/usr/local/lib64"
 	fi
 elif [ "${POLICY}" == "manylinux_2_28" ]; then
-	BASEIMAGE="${MULTIARCH_PREFIX}almalinux:8"
+	BASEIMAGE="almalinux:8"
 	DEVTOOLSET_ROOTPATH="/opt/rh/gcc-toolset-12/root"
 	PREPEND_PATH="${DEVTOOLSET_ROOTPATH}/usr/bin:"
 	LD_LIBRARY_PATH_ARG="${DEVTOOLSET_ROOTPATH}/usr/lib64:${DEVTOOLSET_ROOTPATH}/usr/lib:${DEVTOOLSET_ROOTPATH}/usr/lib64/dyninst:${DEVTOOLSET_ROOTPATH}/usr/lib/dyninst"
+elif [ "${POLICY}" == "manylinux_2_31" ]; then
+	BASEIMAGE="ubuntu:20.04"
+	DEVTOOLSET_ROOTPATH=
+	PREPEND_PATH=
+	LD_LIBRARY_PATH_ARG=
```

Comment:
This commit seems to mix in some changes for 2_31 as well? Should probably be a separate change/PR?

Reply:
The plan is already to split this before setting out of draft.